src/app/actions/tabsLifecycle.ts

```diff
@@ -1,6 +1,7 @@
 import { ReceivedAction } from 'webext/createMessageHandler';
 import Tab from 'app/lmem/tab';
-import { BaseAction, StandardAction, TabAction } from '.';
+import { StandardAction } from 'app/store/types';
```

Comment:
`StandardAction` is only an interface and not a domain related one, it's supposed to be a standard interface, I'd like to avoid mixing it with the application actions.

Reply:
OK I get it